src_driver/pkcs11_CK_ATTRIBUTE.ml

```diff
@@ -144,46 +144,6 @@ let unsafe_get_key_type : t -> Pkcs11_CK_KEY_TYPE.t =
 let unsafe_get_bigint t =
   P11_bigint.decode (unsafe_get_string t)
 
-let decode_ec_point cs =
-  let grammar = Key_parsers.Asn1.EC.point_grammar in
-  let codec = Asn.codec Asn.ber grammar in
-  match Asn.decode codec cs with
```

Comment:
If I understand it correctly, this was the only use of `Key_parsers`. Can you remove it from `_tags`, `opam`, `.merlin` and `META`?